.travis.yml

```diff
@@ -25,7 +25,9 @@ services:
   - redis-server
 
 before_script:
-  - 'RAILS_ENV=test bundle exec rake db:create --trace'
+  #- 'RAILS_ENV=test bundle exec rake db:create --trace'
+  - "createdb travis_test || true"
+  - "curl -fs https://raw.githubusercontent.com/travis-ci/travis-migrations/jc-prefs/db/main/structure.sql | psql travis_test"
```

Comment:
this should be `master` once https://github.com/travis-ci/travis-migrations/pull/138 is merged, right?